lib/vyasa_web/components/contexts/read.ex

```diff
@@ -116,9 +116,9 @@ defmodule VyasaWeb.Context.Read do
       |> stream(:chapters, chapters |> Enum.sort_by(fn chap -> chap.no end))
     else
       [%Chapter{} = chapter | _] ->
-        send(self(), {"helm", ~p"/explore/#{source_title}/#{chapter.no}/"})
 
         socket
+        |> apply_action(:show_verses, params |> Map.put("chap_no", chapter.no))
```

Comment:
@ks0m1c 

would encourage the use of commit message to signpost the bugfix 

e.g. in the message body: 
"this fixes the issue on navigation that existed in the previous use of helm, when we are short-circuiting the nav for sources with a single chapter."